psi4/driver/procrouting/proc.py

```diff
@@ -1299,6 +1300,16 @@ def scf_helper(name, post_scf=True, **kwargs):
     if ref_wfn is not None:
         raise ValidationError("Cannot seed an SCF calculation with a reference wavefunction ('ref_wfn' kwarg).")
 
+    # decide if we keep the checkpoint file
+    _chkfile = kwargs.get('write_orbitals', True)
```

Comment:
Do we want this true by default? For large systems, could the I/O get expensive?

Reply:
Default `True` is the old behaviour, so I kept it. It gets turned off for finite differences currently.
For very large systems (psi4 could handle) the MOs take maybe 1-2GB and one would want a fast disk, but I dont think it would bottleneck soon, especially writing it only at the end of the SCF.